docs/admin/rescheduler.md

```diff
@@ -0,0 +1,58 @@
+---
+assignees:
+- davidopp
+- filipg
+- piosz
+
+---
+
+* TOC
+{:toc}
+
+# "Guaranteed" scheduling of critical addon pods
+
+## Critical addons
+
+In addition to Kubernetes core components like api-server, scheduler, controller-manager running on a master machine
+there are a number of addons which due to various reasons have to run on a regular cluster node, not the master.
+Some of them are critical to have a fully functional cluster: Heapster, DNS, UI.
```

Comment:
"Some of these add-ons are critical to a fully functional cluster, such as Heapster, DNS, and UI."